specs/enhance_gluster_cluster_sync_logic.adoc

```diff
@@ -0,0 +1,531 @@
+= Enhance gluster cluster synchronization logic as per recommendations
+
+The intent of this change is to accommodate additional details available as part
+of `get-state` output while syncing the gluster cluster details and store them
+in central store.
+
+Also instead of using `gstatus` for cluster health and volume utilization
+details, we need to user recommendations provided by gluster team.
+
+
+== Problem description
+
+This specification talks about various changes needed to be done under gluster
+cluster synchronization code to accomodate the additional details related to
+volume snapshots, geo-rep sessions.
+
+Also as `gtstaus` is quite a unstable and also few scenarios are not taken care
+of, using it for deciding the overall cluster status and volume utilization is
+not appropriate. Rather we need to write the aggregations logic to decide the
+cluster state based on peers status and volume status of the gluster trusted
+storage pool. Also the overall volume utilization should be calculated based on
+aggregations done for the participating bricks of the volumes.
+
+
+== Use Cases
+
+This addresses the synchronization of gluster cluster details like
+
+* Volume snapshot details
```

Comment:
If this is about geo-rep objects, why is Volume snapshot details and other redundant data (from #170 ) present here?